packages/api-explorer/scripts/utils.ts

```diff
@@ -26,8 +26,8 @@
 
 import path from 'path'
 import fs from 'fs'
-import { IOauthClientApp, LookerNodeSDK } from '@looker/sdk/lib/node'
-import { NodeSettingsIniFile } from '@looker/sdk-rtl/lib/node'
+import { IOauthClientApp } from '@looker/sdk'
+import { LookerNodeSDK, NodeSettingsIniFile } from '@looker/sdk-rtl-node'
```

Comment:
I don't know how it works to have `LookerNodeSDK` in the `sdk-rtl-node` package but I guess I'll find out further down